database/migrations/2021_09_09_073703_create_messages_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateMessagesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('messages', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id');
+            $table->foreign('user_id')
+                ->references('id')->on('users')
+                ->cascadeOnDelete();
```

Comment:
(suggest)
Laravel 8 から以下のように短く書けます！
see: https://readouble.com/laravel/8.x/ja/migrations.html#foreign-key-constraints
```suggestion
            $table->foreignId('user_id')->constrained()->onDelete('cascade');
```